fe/fe-core/src/main/java/com/starrocks/external/HiveMetaStoreTableUtils.java

```diff
@@ -156,6 +156,9 @@ public static boolean validateColumnType(String hiveType, Type type) {
                         primitiveType == PrimitiveType.VARCHAR;
             case "BOOLEAN":
                 return primitiveType == PrimitiveType.BOOLEAN;
+            case "BINARY":
```

Comment:
`validateColumnType` is used to validate external table schema when creation, so we can just return fail when types are unsupported

Reply:
> `validateColumnType` is used to validate external table schema when creation, so we can just return fail when types are unsupported

All queries related to external hive tables need to validate hive column, so if we return false when table column is unsupported, even if the query has no relation with unsupported type, will fail here.